.bazelrc

```diff
@@ -284,7 +284,7 @@ build:remote-clang-cl --config=rbe-toolchain-clang-cl
 
 # Docker sandbox
 # NOTE: Update this from https://github.com/envoyproxy/envoy-build-tools/blob/main/toolchains/rbe_toolchains_config.bzl#L8
```

Comment:
I check this and find it a long time no update.

I am not sure what kind of process update should I use now.

/cc @phlax @lizan